### PR TITLE
feat: Add HistoricAcceptList param

### PR DESCRIPTION
**Pull request by rbajollari**
## Description

closes: #1598 

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] added `!` to the type prefix if API or client breaking change
- [ ] added appropriate labels to the PR
- [ ] targeted the correct branch (see [PR Targeting](https://github.com/umee-network/umee/blob/main/CONTRIBUTING.md#pr-targeting))
- [ ] provided a link to the relevant issue or specification
- [ ] added a changelog entry to `CHANGELOG.md`
- [ ] included comments for [documenting Go code](https://blog.golang.org/godoc)
- [ ] updated the relevant documentation or specification
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items._

I have...

- [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed state machine logic
- [ ] reviewed API design and naming
- [ ] reviewed documentation is accurate
- [ ] reviewed tests and test coverage
- [ ] manually tested (if applicable)

### CODE REVIEW

File: proto/umee/oracle/v1/oracle.proto [hunk 1]
```diff
@@ -41,16 +41,24 @@ message Params {
     (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Dec",
     (gogoproto.nullable)   = false
   ];
+  // Historic Asset List is a list of assets which will use the historic
+  // price stamping protection methodology (mainly manipulatable assets).
+  // Any assets not on this list will not be stamped.
+  repeated Denom historic_accept_list = 9 [
+    (gogoproto.moretags)     = "yaml:\"historic_accept_list\"",
```
[the remark this answers is not in the document]
Comment (robert-zaremba) [reply proto/umee/oracle/v1/oracle.proto:48]: yes, let's remove this.

File: proto/umee/oracle/v1/oracle.proto [hunk 2]
```diff
@@ -41,16 +41,24 @@ message Params {
...
+    (gogoproto.castrepeated) = "DenomList",
+    (gogoproto.nullable)     = false
+  ];
   // Stamp Period represents the amount of blocks the historacle module
   // waits before recording a set of prices from the oracle.
-  uint64 stamp_period = 9;
```
[the remark this answers is not in the document]
Comment (robert-zaremba) [reply proto/umee/oracle/v1/oracle.proto:46]: :+1:

**Review by robert-zaremba — COMMENTED** [review]
I'm already modifying the whole mechanism of params management, but it will take a bit of time, so it's OK to use legac params object.

**Review by zarazan — APPROVED** [review]
👍🏻

**Pull request merged**